src/Components/FilterComponents/OrderBy.js

```diff
@@ -7,6 +7,7 @@ function OrderBy({ orderColumnRef, orderByRef, fields }) {
         ref={orderColumnRef}
         className="mr-2 border"
         data-testid="data-ord"
+        id="data-ord"
```

Comment:
@tanvirchahal why it is needed?

Reply:
@anuveyatsu refactored code to use existing `data-testid` and removed all new added IDs